src/app/components/arlas-wui-root/arlas-wui-root.component.scss

```diff
@@ -17,14 +17,18 @@
  * under the License.
  */
 
-@import '../../../styles/variables.scss';
-@import '../../../styles/app/sizes.scss';
-@import '../../../styles/app/mixin.scss';
+@import 'styles/variables.scss';
+@import 'styles/app/sizes.scss';
+@import 'styles/app/mixin.scss';
 
```

Comment:
Import like this will not work for the lib build 
(we discussed that this morning with @MohamedHamouGisaia )

Reply:
Indeed, fixed in new version of the commit